Driverless AI/get-started-with-open-source-custom-recipes-tutorial/get-started-with-open-source-custom-recipes-tutorial.md

```diff
@@ -481,27 +500,27 @@ We can see that when we used the Scorer recipe, the model had better AUC with an
 
 H2O custom recipes reside in the H2O Driverless AI Recipes GitHub repo. **There are multiple branches of Driverless AI recipes so make sure that you are using the same branch as the Driverless AI version you have.**
 
-For this tutorial, we are using **Driverless AI 1.8.7.1,** but we will be working off the Driverless AI 1.8.7 branch, as both 1.8.7 and 1.8.7.1 branches have been merged.
+For this tutorial, we are using **Driverless AI 1.9.0,** but we will be working off the Driverless AI 1.9.0 branch.
```

Comment:
This note can be deleted. It was only applicable to the 1.8.7.1 release since there were 2 branches merged for recipes.